lib/riskified_php_sdk/src/Riskified/OrderWebhook/Transport/CurlTransport.php

```diff
@@ -27,6 +27,8 @@ class CurlTransport extends AbstractTransport {
     public $timeout = 10;
     public $dns_cache = true;
 
+    public $requestData;
```

Comment:
why do we need this requestData? 
also please note that this is our php sdk and not the magento extension. there should be a good reason to change the sdk